requirements.txt

```diff
@@ -12,6 +12,8 @@ reportlab==3.5.6
 pdf2image==0.1.14
 PyMuPDF==1.13.20
 Werkzeug==0.14.1  # pyup: < 0.15.0
+pdfrw>=0.4
+defusedxml
```

Comment:
we should pin these deps to versions, the same as above. just stick em both on the latest versions